pystac/item.py

```diff
@@ -71,7 +74,7 @@ def to_dict(self, include_self_link=True):
 
         assets = dict(map(lambda x: (x[0], x[1].to_dict()), self.assets.items()))
 
-        self.properties['datetime'] = '{}Z'.format(self.datetime.replace(microsecond=0, tzinfo=None))
+        self.properties['datetime'] = self.datetime.isoformat()
```

Comment:
This datetime format was intentional, as it matches https://tools.ietf.org/html/rfc3339#section-5.6 per the spec, and matches all the examples of datetime's in the repo.

Reply:
got it